airtable.js

```diff
@@ -0,0 +1,52 @@
+// Don't forget to add your API key to the key variable
+const key = ""
+
+let table = base.getTable("Table 1");
+let view = table.getView('Needs Info');
+let result = await view.selectRecordsAsync({fields: ['Prompt']});
+for (let record of result.records) {
+    if (record) {
+        let prompt = await record.getCellValue('Prompt');
+        output.table(record.getCellValue('Prompt'));
+        output.text(prompt)
+        let results = await chatCompletions(prompt)
+        output.text(results)
+        await table.updateRecordAsync(record, {[`Social post`]: results,});
+    }
+}
+
+async function chatCompletions(prompt) {
```

Comment:
Code organization: It's good to separate concerns and keep the code organized. Consider modularizing the code by placing the chatCompletions function in a separate file or section of the code.